src/models/PostureSession.js

```diff
@@ -20,7 +20,7 @@ const schema = new Schema(
         const startedAt = dayjs(this.started_at);
         const endedAt = dayjs(this.ended_at);
         if (startedAt.isValid() && endedAt.isValid()) {
-          return endedAt.diff(startedAt, "minutes");
+          return endedAt.diff(startedAt, "seconds");
         }
         return 0;
       },
```

Comment:
Yes. When the session is less than a minute, the saved duration value is 0 (minute) which is why we get the 'less than 1 second' from the highlighted section on the screenshot.

For the front end, I tested the Session Summary and the Session History on the Analytics page. It displays by minutes and seconds. Encircled duration are the same Posture Session record.
![image](https://github.com/user-attachments/assets/052969b9-101e-45d7-9cef-c3a2a651fd58)
![image](https://github.com/user-attachments/assets/1bb69fcc-3878-46f2-a65d-5d51af4c32cf)

The drawbacks are:
* Records prior to this change will no longer be correct, because they are based on minutes. 
* Incorrect duration will be displayed on the Session Summary page if the session is more than 1 hour and 59 mins because of the formatting shown below. 😄 
* And I forgot to update the variable name from "minutes" to "seconds" on session-summary.tsx  😅 
![image](https://github.com/user-attachments/assets/1169c822-1062-467c-ba09-78a0b5a068a7)


The easier and safer change is to remove measurements by seconds and all records below 1 minute will be displayed as 'less than 1 minute'.
![image](https://github.com/user-attachments/assets/764c131d-84f3-4ab7-828f-d8754f740e24)

I don't mind implementing the second one. What do you think?

Reply:
Thanks Wonnyo!

I'll just update session-summary.tsx to align the naming convention.